goshopify_test.go

```diff
@@ -40,7 +40,7 @@ func setup() {
 		Scope:       "read_products",
 		Password:    "privateapppassword",
 	}
-	client = NewClient(app, "fooshop", "abcd")
+	client = NewClient(app, "fooshop", "abcd", "2019-04")
```

Comment:
I wonder if we should hard-code the API version in the tests to be an invalid y/d but pass the validation so it's clearly a test value? 0000-00 or 9999-99 or something?

Reply:
Given the pattern change I'm ready to commit the test value would be provided in the NewClient call.  There are a few cases where the value is present in the fixtures, I've set everything at 9999-99